src/AWS.Deploy.CLI/ServerMode/Models/GetConfigSettingResourcesOutput.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.\r
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Collections.Generic;
+
+namespace AWS.Deploy.CLI.ServerMode.Models
+{
+    public class GetConfigSettingResourcesOutput
+    {
+        public Dictionary<string, string>? Resources { get; set; }
```

Comment:
This should be a list of a container resource object. That object should have `DisplayName` and `SystemName` property.

Reply:
Done